### PR TITLE
Add Venetian localization [complete]

### PR DESCRIPTION
**Pull request by xylographe**
PR #7767 extended.

### CODE REVIEW

**Comment by conky77** [comment]
Should I update also Parameters.cpp?

**Comment by xylographe** [comment]
@conky77 
Perhaps, a translation of the 4 missing items:
```xml
<Item subMenuId="language-userDefinedLanguage" name="User Defined Language"/>

<Item id="46300" name="Open User Defined Language folder..."/>

<Item id="6314" name="Use monospaced font in Find dialog (Need to restart Notepad++)"/>

<find-in-files-filter-tip value="Find in cpp, cxx, h, hxx &amp;&amp; hpp:
*.cpp *.cxx *.h *.hxx *.hpp

Find in all files except exe, obj &amp;&amp; log:
*.* !*.exe !*.obj !*.log"/>
```

File: PowerEditor/installer/nativeLang/venetan.xml [hunk 1]
```diff
@@ -0,0 +1,1255 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Venetan translation for Notepad++ 7.8.0
+
+-->
+<NotepadPlus>
+    <Native-Langue name="Venetan" filename="venetan.xml" version="7.8.0">
```
Comment (conky77) [inline PowerEditor/installer/nativeLang/venetan.xml]: <Native-Langue name="Vèneto" filename="venetan.xml" version="7.8.0">

File: PowerEditor/src/localizationString.h [hunk 2]
```diff
@@ -116,5 +116,6 @@ LocalizationSwitcher::LocalizationDefinition localizationDefs[] =
 	{TEXT("Brezhoneg"), TEXT("breton.xml")},
 	{TEXT("کوردی‬"), TEXT("kurdish.xml")},
 	{TEXT("Pig latin"), TEXT("piglatin.xml")},
-	{TEXT("Zulu"), TEXT("zulu.xml")}
+	{TEXT("Zulu"), TEXT("zulu.xml")},
+	{TEXT("Venetan"), TEXT("venetan.xml")}
```
Comment (conky77) [inline PowerEditor/src/localizationString.h]: {TEXT("Vèneto"), TEXT("venetan.xml")}

**Comment by conky77** [comment]
> ```
> <Item subMenuId="language-userDefinedLanguage" name="User Defined Language"/>
> 
> <Item id="46300" name="Open User Defined Language folder..."/>
> 
> <Item id="6314" name="Use monospaced font in Find dialog (Need to restart Notepad++)"/>
> 
> <find-in-files-filter-tip value="Find in cpp, cxx, h, hxx &amp;&amp; hpp:
> *.cpp *.cxx *.h *.hxx *.hpp
> 
> Find in all files except exe, obj &amp;&amp; log:
> *.* !*.exe !*.obj !*.log"/>
> ```

what is the best way to discover the missing translation and to place in the right position. I'm using  the browser to edit it and it is not very easy.

**Comment by xylographe** [comment]
@conky77  Perhaps, [this commit](https://github.com/xylographe/notepad-plus-plus/commit/5a88970c3dc03a0df594af4b41c492b09f344bae) can help?

File: PowerEditor/installer/nativeLang/venetan.xml (hunk 1 above)
Comment (xylographe) [reply PowerEditor/installer/nativeLang/venetan.xml]: Updated.

File: PowerEditor/src/localizationString.h (hunk 2 above)
Comment (xylographe) [reply PowerEditor/src/localizationString.h]: Updated.

File: PowerEditor/installer/nativeLang/venetan.xml [hunk 3]
```diff
@@ -0,0 +1,1255 @@
...
+                    <Item subMenuId="encoding-easternEuropean" name="Eoropa orientaƚe"/>
+                    <Item subMenuId="encoding-greek" name="Greco"/>
+                    <Item subMenuId="encoding-hebrew" name="Ebraico"/>
+                    <Item subMenuId="encoding-japanese" name="Giaponexe"/>
+                    <Item subMenuId="encoding-korean" name="Coreano"/>
+                    <Item subMenuId="encoding-northEuropean" name="Eoropa del nord"/>
+                    <Item subMenuId="encoding-thai" name="Taiƚandese"/>
+                    <Item subMenuId="encoding-turkish" name="Turco"/>
+                    <Item subMenuId="encoding-westernEuropean" name="Eoropa Osidentaƚe"/>
+                    <Item subMenuId="encoding-vietnamese" name="Vietnamita"/>
```
Comment (conky77) [inline PowerEditor/installer/nativeLang/venetan.xml]: <Item subMenuId = "language-userDefinedLanguage" name="Ƚenguagio definìo da 'l utente"/>

File: PowerEditor/installer/nativeLang/venetan.xml [hunk 4]
```diff
@@ -0,0 +1,1255 @@
...
+                    <Item id="45061" name="GB2312 (sempƚificà)"/>
+                    <Item id="45054" name="OEM 861: Isƚandexe"/>
+                    <Item id="45057" name="OEM 865: Nordico"/>
+                    <Item id="45053" name="OEM 860: Portoghexe"/>
+                    <Item id="45056" name="OEM 863: Francexe"/>
+
+                    <Item id="46001" name="Configura i stiƚi..."/>
+                    <Item id="46180" name="Definìo da l&apos;utente"/>
+                    <Item id="46250" name="Definisi el to ƚenguagio..."/>
+
```
Comment (conky77) [inline PowerEditor/installer/nativeLang/venetan.xml]: <Item id="46300" name="Vèrxi ƚa carteƚa de i Ƚenguagi definìi da 'l utente..."/>

File: PowerEditor/installer/nativeLang/venetan.xml [hunk 5]
```diff
@@ -0,0 +1,1255 @@
...
+                    <Item id="6318" name="Link Clicabiƚi"/>
+                    <Item id="6319" name="Abiƚita"/>
+                    <Item id="6320" name="Non sotoƚineare"/>
+
+                    <Item id="6312" name="Riƚevamento Stato File"/>
+                    <Item id="6313" name="Axorna senza notifica"/>
+                    <Item id="6325" name="Vai a l&apos;ùltima riga dopo l&apos;axornamento"/>
+
+                    <Item id="6334" name="Riƚeva automaticamente codifica carateri"/>
```
Comment (conky77) [inline PowerEditor/installer/nativeLang/venetan.xml]: <Item id="6314" name="Uxa font monospasià inte la fenèstra par rumare (cogna aviare de novo Notepad++)"/>

File: PowerEditor/installer/nativeLang/venetan.xml [hunk 6]
```diff
@@ -0,0 +1,1255 @@
...
+Sto file el xe xà stà modificà da &apos;n altro programa.
+Vuto ricaricarƚo?"/>
+            <DoReloadOrNotAndLooseChange title="Ricarica" message="&quot;$STR_REPLACE$&quot;
+Sto file el xe stà modificà da &apos;n altro programa.
+Vuto ricaricarƚo e perdare tute ƚe modifiche aportà in Notepad++?"/>
+            <PrehistoricSystemDetected title="Sistema preistorico trovà" message="Senbra che te si drìo uxare on sistema preistorico. Sta funsionaƚità ƚa necesita de un sistema pì moderno, scusane."/>
+            <XpUpdaterProblem title="Axornamento Notepad++" message="Ƚ&apos;axornamento automatico de Notepad++ no&apos;l xe compatibiƚe con XP a causa de un sistema de permesi obsoƚeto.
+Vuto &apos;ndare ne ƚa pàgina de Notepad++ par scaricare l&apos;axornamento manualmente?"/>
+            <DocTooDirtyToMonitor title="Problema monitoragio" message="El documento el xe sporco. Salva ƚe modifiche prima de aviare el monitoragio."/>
+            <DocNoExistToMonitor title="Probƚema monitoragio" message="El
```
Comment (conky77) [inline PowerEditor/installer/nativeLang/venetan.xml]: <find-in-files-filter-tip value="Ruma en cpp, cxx, h, hxx &amp;&amp; hpp:&#xD;*.cpp *.cxx *.h *.hxx *.hpp&#xD;&#xD;Ruma en tuti i file a ecesion de exe, obj &amp;&amp; log:*.* !*.exe !*.obj !*.log"/>

**Comment by xylographe** [comment]
@donho 
All occurrences of  **`[Vv]enetan`** replaced with **`[Vv]enitian`**.

**Pull request closed without merging**